app/routes/helper_functions.py

```diff
@@ -0,0 +1,296 @@
+from flask import make_response, request, abort, jsonify
+from app.models.task import Task
+from sqlalchemy import asc, desc
+from dotenv import load_dotenv
+from datetime import datetime
+from http import HTTPStatus
+from app import db
+import requests
+import os
+
+
+load_dotenv()
+
+
+NOWTIME = datetime.utcnow()
+SLACK_TOKEN = os.environ.get("SLACK_TOKEN")
+CHANNEL = os.environ.get("CHANNEL")
+
+
+def create_response(cls, instance, status_code=HTTPStatus.OK):
+    db.session.commit()
+    instance = instance.to_json()
+    cls_type = cls.__name__.lower()
+    return make_response({cls_type: instance}, status_code)
+
+
+def generate_error_message(cls, id):
+    return {"message": f"{cls.__name__} {id} was not found."}
+
+
+def validate_id(cls, id):
+    try:
+        id = int(id)
+    except ValueError:
+        error_message = generate_error_message(cls, id)
+        abort(make_response(error_message, HTTPStatus.BAD_REQUEST))
+    return id
+
+
+def get_model_by_id(cls, id):
+    id = validate_id(cls, id)
+    model = db.session.query(cls).get(id)
+
+
+    if not model:
+        error_message = generate_error_message(cls, id)
+        abort(make_response(error_message, HTTPStatus.NOT_FOUND))
```

Comment:
Great error handling. However, this particular check should already be handled by your validate_id method so it's not necessary in this context!